server/app/assets/javascripts/applicant_entry_point.ts

```diff
@@ -25,4 +26,5 @@ window.addEventListener('load', () => {
   azureDelete.init()
   azureUpload.init()
   phoneNumber.init()
+  apiDocs.init()
```

Comment:
suggestion (not blocking): would it be helpful to add a comment here explaining why the api docs are in the applicant bundle?